doc/classes/CharacterBody2D.xml

```diff
@@ -73,8 +73,8 @@
 				[codeblocks]
 				[gdscript]
 				for i in get_slide_collision_count():
-				var collision = get_slide_collision(i)
-				print("Collided with: ", collision.collider.name)
+					var collision = get_slide_collision(i)
+					print("Collided with: ", collision.get_collider().name)
```

Comment:
Looks correct, but we use 4-spaces for indentation within codeblocks. Could you amend the commit to change that?

Reply:
Done!